From ee61f268bd23f3da552d8953df97453769da7a36 Mon Sep 17 00:00:00 2001
From: Bert Geens <bert@lair.be>
Date: Tue, 21 Jan 2014 17:39:19 +0100
Subject: [PATCH 1/1] Fix NPE in "Selected Module Last Version"

Fix NPE caused by the Jenkins Job name being different from the actual Maven
artifactId.
---
 .../java/jenkins/plugins/maveninfo/columns/LastVersionColumn.java   | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/src/main/java/jenkins/plugins/maveninfo/columns/LastVersionColumn.java b/src/main/java/jenkins/plugins/maveninfo/columns/LastVersionColumn.java
index 47eb1bf..39e5066 100644
--- a/src/main/java/jenkins/plugins/maveninfo/columns/LastVersionColumn.java
+++ b/src/main/java/jenkins/plugins/maveninfo/columns/LastVersionColumn.java
@@ -101,11 +101,7 @@ public class LastVersionColumn extends AbstractMavenInfoColumn {
 	}
 
 	public String getVersion(MavenModuleSet job) {
-		MavenModuleSetBuild build = getBuild(job);
-		ModuleNamePattern mainPattern = getModulePattern(job);
-
-		MavenModule m = BuildUtils.getMainModule(build, mainPattern);
-		return m.getVersion();
+		return job.getRootModule().getVersion();
 	}
 
 	public boolean isMultipleVersions(MavenModuleSet job) {
-- 
1.8.3.2

